ui/components/ui/toggle-button/toggle-button.component.js

```diff
@@ -47,15 +47,22 @@ const colors = {
 };
 
 const ToggleButton = (props) => {
-  const { value, onToggle, offLabel, onLabel } = props;
+  const { value, onToggle, offLabel, onLabel, disabled } = props;
 
   const modifier = value ? 'on' : 'off';
 
+  // eslint-disable-next-line no-empty-function
+  const noop = () => {};
+
   return (
-    <div className={classnames('toggle-button', `toggle-button--${modifier}`)}>
+    <div
+      className={classnames('toggle-button', `toggle-button--${modifier}`, {
+        'toggle-button--disabled': disabled,
+      })}
+    >
       <ReactToggleButton
         value={value}
-        onToggle={onToggle}
+        onToggle={disabled ? noop : onToggle}
```

Comment:
Nit: we can pass in `undefined` here instead - the ReactToggleButton prop doesn't require us passing in a function.